argocd.tf

```diff
@@ -1,252 +1,282 @@
 ### ArgoCD helm
 locals {
```

Comment:
Can we have an option to fully override helm values if so needed on the external side? For example, I have already helm chart deployment, and I just want to use my values, but want to move management of it to this module - would this be considered a scope of this module, potentially?

Reply:
Maybe in the future, but not at the moment, I think